test/envoye2e/stats_plugin/stats_test.go

```diff
@@ -61,11 +61,11 @@ var Runtimes = []struct {
 	},
 	// https://issues.redhat.com/browse/MAISTRA-2140
 	// FIXME: Reenable when the issue above is fixed
-	// {
```

Comment:
These changes are not related, see the FIXME line above.

Reply:
No, the network was open

Reply:
Build to be restarted with that options and without modifications in stats_test.go